themes/default/content/blog/announcing-public-preview-update-plans/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Announcing the public preview of Update Plans"
+date: 2022-02-10T08:00:00-07:00 # TODO update this
+meta_desc: Update Plans enable you to see and confirm the updates that will be made to your infrastructure and then apply those updates at a later time.
+meta_image: meta.png # TODO update this
+authors:
+    - fraser-waters
+tags:
+    - features
+---
+
+## Announcing the public preview of Update Plans
+
+Pulumi’s previews are an important part of any workflow where you want to see the changes that will be made to your infrastructure before actually making the changes (with `pulumi up`). However, today there is no guarantee that the `pulumi up` operation will do only what was previewed; if the program, or your infrastructure, changes between the preview and the update, the update might make additional changes to bring your infrastructure back in line with what’s defined in your program. We’ve [heard from many of you](https://github.com/pulumi/pulumi/issues/2318) that you need a strong guarantee about exactly which changes an update will make to your infrastructure, especially in critical and production environments.
```

Comment:
> your infrastructure, changes between the preview and the update

Do update plans fix this? (drift of infra) I guess it might - in combination with refresh?